### DISCUSSION TITLE
Context Menu Plugin /  Directive

### DISCUSSION THREAD

**Discussion by gethari** [sibiraj-s/ngx-editor#427] [Ideas]
Hi @sibiraj-s , I have recently created a context menu directive to render any angular component inside it, similar to the bubble menu.

![image](https://user-images.githubusercontent.com/20477956/183822856-d416fa4f-94db-463a-8bea-3dd5d62adad5.png)

When I right click on the table, this is how it looks. I mostly put together the `context-menu.directive` similar to how you implemented it for bubble menu. Thank you for that 👏 

Do you want me to raise a PR so that it can be included in the lib ?

**Comment by sibiraj-s** [comment]
> Do you want me to raise a PR so that it can be included in the lib ?

Sure. Go for it. And thanks 🔥

[a comment by an automated account is not shown]
